src/settings/BursarExports/apiQuery.js

```diff
@@ -0,0 +1,61 @@
+import { useMutation, useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { SCHEDULE_PERIODS } from './BursarExportsConfiguration';
+
+const bursarConfigApi = 'batch-voucher/export-configurations';
+
+export const useBursarConfigQuery = (key = 'bursarConfig') => {
+  const ky = useOkapiKy();
+
+  const { isLoading, data = {} } = useQuery({
+    queryKey: key,
+    queryFn: async () => {
+      const { records = [] } = await ky.get(bursarConfigApi).json();
+
+      return records[0] || {
+        schedulePeriod: SCHEDULE_PERIODS.none,
+      };
+    },
+    retry: false,
```

Comment:
In default config retry is [set to false](https://github.com/folio-org/stripes-core/blob/master/src/createReactQueryClient.js#L9), so I believe this one is not needed.